modules/oe_webtools_analytics/src/Form/WebtoolsAnalyticsSettingsForm.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_analytics\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Settings form for module.
+ */
+class WebtoolsAnalyticsSettingsForm extends ConfigFormBase {
+
+  /**
+   * Name of the config being edited.
+   */
+  const CONFIGNAME = 'oe_webtools_analytics.settings';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'oe_webtools_analytics_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
```

Comment:
I'd move the protected methods to the bottom.

First the public, then the protected, then the private.